.github/workflows/release.yml

```diff
@@ -6,17 +6,14 @@
 
 name: Release
 on:
+  workflow_dispatch:
   push:
     # Path filters are not evaluated for pushes to tags.
     # (source: https://help.github.com/en/github/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#onpushpull_requestpaths)
     # So this workflow is triggered in the following events:
     # - Release event: a SemVer tag, e.g. v1.0.0 or v1.0.0-alpha, is pushed
     tags:
       - 'v[0-9]+.[0-9]+.[0-9]+*'
-    # OR
-    # - Test event: this file is modified on a branch in the main repo containing `/actions/` in the name.
-    branches:
-      - '**/actions/**'
     paths:
       - '**/workflows/release.yml'
```

Comment:
Sorry, realised this `paths` needs to be removed along with the `branches`, OR even better, we turn it into a `pull_request: paths: '**/workflows/release.yml'`. Then we can test PRs that change the workflow itself without running on every push.

Reply:
Yep. That makes sense.